test/e2e/framework/volume/fixtures.go

```diff
@@ -156,6 +157,9 @@ func NewNFSServer(cs clientset.Interface, namespace string, args []string) (conf
 		config.ServerArgs = args
 	}
 	pod, ip = CreateStorageServer(cs, config)
+	if strings.Contains(ip, ":") {
```

Comment:
this is the "same" that net.JoinHostPort is doing

Reply:
seems wrong to return this as an "ip", maybe s/ip/host?

Reply:
or make the caller consume the ip correctly?

Reply:
returning host instead of ip is easier and I think that is ok